modules/custom/activity_creator/src/ActivityNotifications.php

```diff
@@ -41,11 +66,14 @@ public function getNotifications(AccountInterface $account, array $status = [ACT
    *
    * @return array
    *   Return array of notifications as activity objects.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
    */
-  public function getNotificationsActivities(AccountInterface $account, array $status = [ACTIVITY_STATUS_RECEIVED]) {
+  public function getNotificationsActivities(AccountInterface $account, array $status = [ACTIVITY_STATUS_RECEIVED]): array {
     $ids = $this->getNotificationIds($account, $status);
 
-    return entity_load_multiple('activity', $ids);
+    return \Drupal::entityTypeManager()->getStorage('activity')->loadMultiple($ids);
```

Comment:
This still needs dependency injection.

Reply:
I will use entityManager() from ControllerBase class.